src/library_manager.py

```diff
@@ -896,6 +899,52 @@ def update(self, sg, deficiency=0):
                 self.sg_list[sg] = self.sg_list[sg]+1
             Trace.trace(self.trace_level, "postponed update %s %s %s"%(sg, deficiency, self.sg_list[sg]))
 
+class PostponedBoundRequests:
+    """
+    Postponed requests from movers with bound volumes. 
+    Requests get put into this "list" because they came while the mover thread was alredy running.
+
+    """
+    def __init__(self):
+        """
+        :type keep_time: :obj:`int`
+        :arg keep_time: maximum time interval to keep request in seconds
+        """
+        self.rq_list = [] # request list 
+        self.trace_level = 340
+
+    def __repr__(self):
+        return 'rq_list:%s'%(self.rq_list,)
```

Comment:
think PEP8 requires spaces around '%'

Reply:
Checked PEP8 and did not find any recommendations regarding this style.